### PR TITLE
Fix duplicate actions; Track notified accounts per action

### PR DESCRIPTION
**Pull request by flux627**
Since there is an entry in the `action_traces` MongoDB collection per notified account, we have to de-duplicate actions from this collection with the same action digest. Additionally, we are now exposing which accounts are notified from the action payload via `Action.payload.notifiedAccounts`. Since this is an optional property, this is not a breaking change.

### CODE REVIEW

File: src/MongoBlock.ts [hunk 1]
```diff
@@ -19,11 +19,17 @@ export class MongoBlock implements Block {
     const eosActions = []
     let currentTx = ""
     let actionIndex = 0
+    let currentActionDigest = ""
     for (const rawAction of rawActions) {
-      if (rawAction.trx_id != currentTx) {
+      if (rawAction.trx_id !== currentTx) {
```
Comment (chris-allnutt) [inline src/MongoBlock.ts]: is there ever a time when action traces won't be sequential?  Or is it guaranteed they'll always be in sequence?

Comment (wpwood) [inline src/MongoBlock.ts]: Need a lint rule to catch this.

Comment (flux627) [reply src/MongoBlock.ts]: This detects when actions transition from one transaction to another. They way they're ordered is grouped, so when we see the tx_id change, we know we're in a new transaction.

Comment (flux627) [reply src/MongoBlock.ts]: To answer your question... this currently depends on insertion order, I believe. But, insertion order will always be by global action sequence, due to how the MongoDB plugin needs to operate.

**Pull request merged**